src/test/java/org/springframework/data/neo4j/integration/imperative/RepositoryIT.java

```diff
@@ -3151,18 +3140,24 @@ void shouldFindExtendedNodeViaExtendedAttribute(@Autowired ParentRepository repo
 		}
 	}
 
-	interface BidirectionalStartRepository extends Neo4jRepository<BidirectionalStart, Long> {}
+	interface BidirectionalStartRepository extends Neo4jRepository<BidirectionalStart, Long> {
```

Comment:
Our formatter config is at https://github.com/spring-projects/spring-data-build/blob/master/etc/ide/eclipse-formatting.xml. Let me know whether the formatting changes here stem from our config so we can fix that on our side.

Reply:
I thought we had this applied. My bad if that isn't the case. 

Regarding the star imports: Isn't the common agreement that they should be avoided? At least that was my impression when I did this: 7804bdcd150b26fadb849ba55da1977d148967c5

Reply:
I think your code is fine, I assume some IDE settings or an outdated config on our side.

RE star imports: Each Spring project has its own approach to code style. Following the Boot style is fine, it would be good to document that somewhere as the Spring Data import style looks different.

Reply:
I am a bit surprised that Jenkins is happy… I just tracked the branch and a `mvn checkstyle:check@validate` fails as expected. Checkstyle runs in `verify` phase, after all tracks (as we discussed a while back in d824e5d4327ca15c49d1b26053ebb9583e1a027c).

Reply:
Actually, the associated Job fails (https://jenkins.spring.io/view/SpringData/job/spring-data-neo4j/job/issue%252FDATAGRAPH-1428/4/). It's green because we suppress upstream trigger causes. For some reason, the suppression still creates a no-op job that appears to be green.

Reply:
I didn't have the proper code formatting applied. I fixed that but still have some changes left. I'll undo them manually.

- Spring Data Formatting prescribes star imports for static imports and for 10 or more imports.
- The import order seams to be subtly different, the ordering of subpackages is different.